libraries/chain/finality/finalizer.cpp

```diff
@@ -305,4 +323,16 @@ void my_finalizers_t::set_default_safety_information(const fsi_t& fsi) {
    default_fsi = fsi;
 }
 
+bool  my_finalizers_t::are_equivalent(uint32_t version, const fsi_map& old, const fsi_map& current) {
+   assert(version < current_safety_file_version); // this function compares an older version with the current one
+   switch(version) {
```

Comment:
Why do we need `version`, as it is hardcoded to `0` anyway?

Reply:
> Why do we need `version`, as it is hardcoded to `0` anyway?

Where is it hardcoded to zero? It can take all values less than `current_safety_file_version`

Reply:
I meant the `case 0:`.  OK, currently `current_safety_file_version` is 1.